src/Component/People.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import axios from 'axios';
+
+class People extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            search: "",
+            people: [],
+            personName: '',
+            personAge: 0,
+            personGender: '',
+            personId: '',
+            isError: false,
+            dispose: '',
+
+        }
+    }
+
+    personSearch = async () => {
+        try {
+            const { data } = await axios.get(`https://ghibliapi.herokuapp.com/people/`)
+            this.setState({ people: data })
+            const newArray = data.filter((p) => {
```

Comment:
You can use .find instead of .filter. It will return the first element that matches the condition. Then on line 28, you don't have iterate through an array.